src/query/v2/interpret/awesome_memgraph_functions.hpp

```diff
@@ -31,7 +34,9 @@ const char kId[] = "ID";
 }  // namespace
 
 struct FunctionContext {
-  DbAccessor *db_accessor;
+  // TODO(kostasrim) consider optional here. ShardRequestManager does not exist on the storage.
```

Comment:
But both storage and query engine can provide a `NameIdMapper` which is actually necessary for the accessors!